src/main/java/io/jenkins/plugins/PurgeBuildQueueAction.java

```diff
@@ -57,7 +57,7 @@ private static String getItemsVerbage(final int length) {
 
     @RequirePOST
     public void doPurge(final StaplerRequest request, final StaplerResponse response) throws ServletException, IOException {
-        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        Jenkins.get().checkPermission(Jenkins.MANAGE);
```

Comment:
Switch to `MANAGE` as this does not allow arbirary command execution or esclation of privalledges